lab_5_scrapper/scrapper.py

```diff
@@ -212,15 +212,15 @@ def find_articles(self) -> None:
                 if len(self.urls) >= self._config.get_num_articles():
                     return
                 url = self._extract_url(paragraph)
-                if not url or url in self.urls:
+                if not url or url in self.urls or url == "url not found":
                     continue
                 self.urls.append('https://krsk.sibnovosti.ru'+ url)
 
     def get_search_urls(self) -> list:
         """
         Returns seed_urls param
         """
-        return self._seed_urls
+        return self._config.get_seed_urls()
```

Comment:
you do not need to get seed urls from config inside this method 

it is used to get self.urls parameter